extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -514,6 +520,16 @@ export class InstrumentationService {
     this.win_.setTimeout(this.win_.clearInterval.bind(this.win_, intervalId),
         maxTimerLength * 1000);
   }
+
+  isTriggerAllowed_(triggerType, element) {
+    if (!isInIframe(element) ||
```

Comment:
Yeah, looked at that tag is probably the right thing (via `document-info.js`). Otherwise something like `viewer.isIframe() && !viewer.isEmbedded()` would work, but be very confusing and brittle.